spec/features/user_sign_in_spec.rb

```diff
@@ -0,0 +1,20 @@
+feature 'User sign in' do
+  let(:user) do
+     User.create(email: 'user@example.com',
+                 password: 'secret1234',
+                 password_confirmation: 'secret1234')
+     end
+
+    def sign_in email:, password:
```

Comment:
Unused method argument - `email`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `password`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.<br>Inconsistent indentation detected.